dependencies/README.md

```diff
@@ -68,3 +77,14 @@ Entry point classes are different for the input of checks:
   - **Check for a list of Maven coordinates**: classes in the Maven artifacts
   - **Check for a list of class and jar files**: all classes in the input are entry points.
     This means that every linkage error is reachable.
+
+### Exclusion Files
+
+Users can specify an exclusion file to filter out known linkage errors that we know are not harmful.
```

Comment:
delete "that we know are not harmful".
In general, Google style avoids "we" (first person plural)

Reply:
Updated